ansible/roles/vendor/gsa.datagov-deploy-jumpbox/meta/main.yml

```diff
@@ -0,0 +1,14 @@
+galaxy_info:
+  author: "GSA"
+  description: "Ansible role to apply configure users on jumpbox"
+  company: GSA
+  license: MIT
```

Comment:
Any work we do is CCO Public Domain

Reply:
This is [fixed upstream](https://github.com/GSA/datagov-deploy-jumpbox/pull/1/files#diff-9b4ddf055111f4371f04fbb8b5366655R6), it'll get pulled in on the next version bump.